integration-tests/run_cluster.sh

```diff
@@ -218,10 +188,44 @@ fi
     docker exec -t druid-it-hadoop sh -c "./usr/local/hadoop/bin/hdfs dfs -chmod -R 777 /user"
     # Copy data files to Hadoop container
     docker exec -t druid-it-hadoop sh -c "./usr/local/hadoop/bin/hdfs dfs -put /shared/wikiticker-it/wikiticker-2015-09-12-sampled.json.gz /quickstart/wikiticker-2015-09-12-sampled.json.gz"
+    docker exec -t druid-it-hadoop sh -c "./usr/local/hadoop/bin/hdfs dfs -put /resources/data/batch_index /batch_index"
     echo "Finished setting up druid hadoop dirs"
 
     echo "Copying Hadoop XML files to shared"
     docker exec -t druid-it-hadoop sh -c "cp /usr/local/hadoop/etc/hadoop/*.xml /shared/hadoop_xml"
     echo "Copied Hadoop XML files to shared"
   fi
+
+  # Start zookeeper and kafka
```

Comment:
Moved all these to after starting Hadoop since we need Hadoop XML files to be copied to shared before starting Druid